src/cordova/exoplayer.js

```diff
@@ -348,9 +348,9 @@ define(['events', 'appSettings', 'filesystem', 'loading'], function (events, app
                                 Context: 'Streaming'
                             },
                             {
-                                Container: 'mp3',
+                                Container: 'ogg',
                                 Type: 'Audio',
-                                AudioCodec: 'mp3',
+                                AudioCodec: 'opus',
```

Comment:
Does this set the preferred codec to OPUS for audio transcodes? We definitely want to keep ACC or MP3 since those formats are more universal.

Reply:
Well exo player supports those (so there's no problem with client compatibility) and they offer better quality to they are a far better choice for transcoding (as the goal of transcoding is to convert to a supported format or to a lower bitrate while preserving as much quality as possible (which mp3 is clearly the worst choice for)).

Reply:
"Opus replaces both Vorbis and Speex for new applications, and several blind listening tests have ranked it higher-quality than any other standard audio format at any given bitrate until transparency is reached, including MP3, AAC, and HE-AAC." https://en.m.wikipedia.org/wiki/Opus_(audio_format)

Reply:
Here's the basis of this claim: http://listening-test.coresv.net/results.htm

Reply:
That's a good enough reason for me, thanks for the links.